.github/dockerfiles/ov_build/ubuntu_22_04_x64_cc/Dockerfile

```diff
@@ -68,10 +62,11 @@ RUN mkdir ${SCCACHE_HOME} && cd ${SCCACHE_HOME} && \
 
 # Setup pip
 ENV PIP_VERSION="24.0"
-RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
-    python3.8 get-pip.py  --no-cache-dir pip==${PIP_VERSION} && \
+RUN curl https://bootstrap.pypa.io/pip/3.8/get-pip.py -o get-pip-3.8.py && \
```

Comment:
@ilya-lavrenov 
FYI
I also had to update this part, since latest `get-pip.py` version requires minimum python3.9